**Materials and Method**

 **Prediction of Phytochemical Properties :**

Activity-based predicted peptides were optimized to enhance desired biological activities. Optimized peptides were exported in FASTA format for subsequent bioinformatic analysis. To predict various physicochemical properties, we employed two online tools: APD3: Antimicrobial Peptide Database and ExPASy ProtParam.

APD3 was used to determine the hydrophobic amino acid content, glycine and proline residue counts, the number of negatively and positively charged amino acids, other amino acid composition, total hydrophobic ratio, net charge at physiological pH, molecular weight, Wimley hydrophobicity, and Boman index, a measure of potential protein-binding affinity.

ExPASy ProtParam was used to predict the theoretical isoelectric point, molecular formula, atomic composition, total number of atoms, estimated half-life in mammalian cells, instability index, aliphatic index, GRAVY score, and extinction coefficient. Peptides with instability index values below 40 were considered stable.

**Table 1: Predicted  Physicochemical Properties using APD3 Database**

|  Peptide ID |  Metabolite (peptide) |  Hydrophobic amino acid |  |  |  |  |  |  |  | The number of G and P |  | Negatively charged amino acid |  | Positively charged amino acid |  |  | Other amino acids |  |  |  |  | total hydrophobic ratioÂ (%) | The total net charge | Molecular Weight (MW) | WIMELY HYDROPHOBICITY | Boman index (kcal/mol) |  |
| ----- | :---- | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | ----- | ----- |
|  |  | **I** | **V** | **L** | **F** | **C** | **M** | **A** | **W** | **G** | **P** | **E** | **D** | **K** | **R** | **H** | **T** | **S** | **Y** | **Q** | **N** |  |  |  |  |  |  |
| **PD101** | [GYPVMIKAALGGGGRGMRVA](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=GYPVMIKAALGGGGRGMRVA&thval=0.0)  | 1 | 2 | 1 | 0 | 0 | 2 | 3 | 0 | 6 | 1 | 0 | 0 | 1 | 2 | 0 | 0 | 0 | 1 | 0 | 0 | 45 | 3 | 1961.387 | 1.5 | 0.09 |  |
| **PD102** | FYPVMIKAALGGGGRGMRVA | 1 | 2 | 1 | 1 | 0 | 2 | 3 | 0 | 5 | 1 | 0 | 0 | 1 | 2 | 0 | 0 | 0 | 1 | 0 | 0 | 50 | 3 | 2051.512 | 0.36 | \-0.01 |  |
| **PD103** | FEPVRLEACKGGGGRGMRVA | 0 | 2 | 1 | 1 | 1 | 1 | 2 | 0 | 5 | 1 | 2 | 1 | 1 | 3 | 0 | 0 | 0 | 0 | 0 | 0 | 40 | 2 | 2090.463 | 6.28 | 1.79 |  |
| **PD104** | LLSELGIKKSDKVIIFKSSA | 3 | 1 | 3 | 1 | 0 | 0 | 1 | 0 | 1 | 0 | 1 | 1 | 4 | 0 | 0 | 0 | 4 | 0 | 0 | 0 | 45 | 2 | 2176.613 | 4.24 | 0.6 |  |
| **PD105** | LLSELGEKKSDKVIIFKSSA | 2 | 1 | 3 | 1 | 0 | 0 | 1 | 0 | 1 | 0 | 2 | 1 | 4 | 0 | 0 | 0 | 4 | 0 | 0 | 0 | 40 | 1 | 2192.569 | 6.57 | 1.18 |  |
| **PD106** | LRFVCGEKCKDKVIIFKSSA | 2 | 2 | 1 | 2 | 2 | 0 | 1 | 0 | 1 | 0 | 1 | 1 | 4 | 1 | 0 | 0 | 2 | 0 | 0 | 0 | 50 | 3 | 2271.765 | 4.68 | 1.26 |  |
| **PD107** | IVFNIKKEITKGILDVGDKI | 5 | 2 | 1 | 1 | 0 | 0 | 0 | 0 | 2 | 0 | 1 | 2 | 4 | 0 | 0 | 1 | 0 | 0 | 0 | 1 | 45 | 1 | 2243.705 | 5.92 | 0.65 |  |
| **PD108** | IEFNIKKEITKGILDVGDKI | 5 | 1 | 1 | 1 | 0 | 0 | 0 | 0 | 2 | 0 | 2 | 2 | 4 | 0 | 0 | 1 | 0 | 0 | 0 | 1 | 40 | 0 | 2273.688 | 7.87 | 1.2 |  |
| **PD109** | FEFACEEECTKGILDVGDKI | 2 | 1 | 1 | 2 | 2 | 0 | 1 | 1 | 2 | 0 | 4 | 2 | 2 | 0 | 0 | 1 | 0 | 0 | 0 | 0 | 45 | \-4 | 2246.542 | 9 | 1.36 |  |
| **PD110** | AAIAFHTPYTKMGKKALLPM | 1 | 0 | 2 | 1 | 0 | 2 | 4 | 0 | 1 | 2 | 0 | 0 | 3 | 0 | 1 | 2 | 0 | 1 | 0 | 0 | 50 | 3.25 | 2189.709 | 1.05 | \-0.2 |  |
| **PD111** | [AAIAFHTCYTKMGKKALLPM](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=AAIAFHTCYTKMGKKALLPM&thval=0.0) | 1 | 0 | 2 | 1 | 1 | 2 | 4 | 0 | 1 | 1 | 0 | 0 | 3 | 0 | 1 | 2 | 0 | 1 | 0 | 0 | 55 | 3.25 | 2195.737 | 0.36 | \-0.26 |  |
| **PD112** | [AEPCCETCCTKMGKKALLPM](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=AEPCCETCCTKMGKKALLPM&thval=0.0) | 0 | 0 | 2 | 0 | 4 | 2 | 2 | 0 | 1 | 2 | 2 | 0 | 3 | 0 | 0 | 2 | 0 | 0 | 0 | 0 | 50 | 1 | 2157.709 | 6 | 0.55 |  |
| **PD113** | [ALLLLKQVVKMCYFNNKVTK](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=ALLLLKQVVKMCYFNNKVTK&thval=0.0) | 0 | 3 | 4 | 1 | 1 | 1 | 1 | 0 | 0 | 0 | 0 | 0 | 4 | 0 | 0 | 1 | 0 | 1 | 1 | 2 | 55 | 4 | 2353.954 | 1.12 | 0.17 |  |
| **PD114** | [ALLLLKQSVKMCYFNNKVTK](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=ALLLLKQSVKMCYFNNKVTK&thval=0.0) | 0 | 2 | 4 | 1 | 1 | 1 | 1 | 0 | 0 | 0 | 0 | 0 | 4 | 0 | 0 | 1 | 1 | 1 | 1 | 2 | 50 | 4 | 2341.899 | 1.18 | 0.54 |  |
| **PD115** | [KLLVVLGSVVMCYFNNKVTK](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=KLLVVLGSVVMCYFNNKVTK&thval=0.0) | 0 | 5 | 3 | 1 | 1 | 1 | 0 | 0 | 1 | 0 | 0 | 0 | 3 | 0 | 0 | 1 | 1 | 1 | 0 | 2 | 55 | 3 | 2255.81 | 0.22 | \-0.32 |  |
| **PD116** | [AIKDIEVEVDLSLFQRSSKG](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=AIKDIEVEVDLSLFQRSSKG&thval=0.0) | 2 | 2 | 2 | 1 | 0 | 0 | 1 | 0 | 1 | 0 | 2 | 2 | 2 | 1 | 0 | 0 | 3 | 0 | 1 | 0 | 40 | \-1 | 2234.532 | 7.71 | 1.96 |  |
| **PD117** | [AEKDIEVEVDLSLFQRSSKG](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=AEKDIEVEVDLSLFQRSSKG&thval=0.0) | 1 | 2 | 2 | 1 | 0 | 0 | 1 | 0 | 1 | 0 | 3 | 2 | 2 | 1 | 0 | 0 | 3 | 0 | 1 | 0 | 35 | \-2 | 2250.488 | 10.04 | 2.55 |  |
| **PD118** | [FEPCIEEEVCLSLFQRSSKG](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=FEPCIEEEVCLSLFQRSSKG&thval=0.0) | 1 | 1 | 2 | 2 | 2 | 0 | 0 | 0 | 1 | 1 | 4 | 0 | 1 | 1 | 0 | 0 | 3 | 0 | 1 | 0 | 40 | \-2 | 2301.628 | 7.21 | 1.75 |  |
| **PD119** | [AVRGKGTFESAIKSVALLVK](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=AVRGKGTFESAIKSVALLVK&thval=0.0) | 1 | 3 | 2 | 1 | 0 | 0 | 3 | 0 | 2 | 0 | 1 | 0 | 3 | 1 | 0 | 1 | 2 | 0 | 0 | 0 | 50 | 3 | 2074.487 | 4.38 | 0.52 |  |
| **PD120** | [AVRGCGTFESAIKSVALLVK](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=AVRGCGTFESAIKSVALLVK&thval=0.0) | 1 | 3 | 2 | 1 | 1 | 0 | 3 | 0 | 2 | 0 | 1 | 0 | 2 | 1 | O | 1 | 2 | 0 | 0 | 0 | 55 | 2 | 2049.462 | 3.15 | 0.18 |  |
| **PD121** | [ARFCCETECSAIKSVALLVK](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=ARFCCETECSAIKSVALLVK&thval=0.0) | 1 | 2 | 2 | 1 | 3 | 0 | 3 | 0 | 0 | 0 | 2 | 0 | 2 | 1 | 0 | 1 | 2 | 0 | 0 | 0 | 60 | 1 | 2171.631 | 4.6 | 0.69 |  |
| **PD122** | [LQRSVGHISQKVLTQHLRAL](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=LQRSVGHISQKVLTQHLRAL&thval=0.0) | 1 | 2 | 4 | 0 | 0 | 0 | 1 | 0 | 1 | 0 | 0 | 0 | 1 | 2 | 2 | 1 | 2 | 0 | 3 | 0 | 40 | 3.5 | 2284.694 | 2.86 | 1.76 |  |
| **PD123** | [LQRSVGHISKKVLTQHLRAL](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=LQRSVGHISKKVLTQHLRAL&thval=0.0) | 1 | 2 | 4 | 0 | 0 | 0 | 1 | 0 | 1 | 0 | 0 | 0 | 2 | 2 | 2 | 1 | 2 | 0 | 2 | 0 | 40 | 4.5 | 2284.733 | 3,27 | 1.76 |  |
| **PD124** | [LQPCCEECSKKVLTQHLRAL](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=LQPCCEECSKKVLTQHLRAL&thval=0.0) | 0 | 1 | 4 | 0 | 3 | 0 | 1 | 0 | 0 | 1 | 2 | 0 | 2 | 1 | 1 | 1 | 1 | 0 | 2 | 0 | 45 | 1.25 | 2299.765 | 6.16 | 1.59 |  |
| **PD125** | [VLCIGYGSKKCIGLGTSTQI](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=VLCIGYGSKKCIGLGTSTQI&thval=0.0) | 3 | 1 | 2 | 0 | 2 | 0 | 0 | 0 | 4 | 0 | 0 | 0 | 2 | 0 | 0 | 2 | 2 | 1 | 1 | 0 | 45 | 2 | 2041.459 | \-0.26 | \-0.31 |  |
| **PD126** | [VLCIGEGSKKCIGLGTSTQI](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=VLCIGEGSKKCIGLGTSTQI&thval=0.0) | 3 | 1 | 2 | 0 | 2 | 0 | 0 | 0 | 4 | 0 | 1 | 0 | 2 | 0 | 0 | 2 | 2 | 0 | 1 | 0 | 40 | 1 | 2007.399 | 2.7 | 0.02 |  |
| **PD127** | [KEGCCEGSKKCIGLGTSTQI](https://webs.iiitd.edu.in/raghava/anticp/pepsearch1.php?seq=KEGCCEGSKKCIGLGTSTQI&thval=0.0) | 2 | 0 | 1 | 0 | 3 | 0 | 0 | 0 | 4 | 0 | 2 | 0 | 3 | 0 | 0 | 2 | 2 | 0 | 1 | 0 | 30 | 1 | 2042.377 | 6.27 | 1.26 |  |

**Table 2:**  **Predicted  Physicochemical Properties using ExPASy ProtParam Database**

| Peptide ID | Theoretical pI | Formula | Atomics Composition |  |  |  |  | Total NO. of Atoms | Estimated half-life |  |  | Instability index (II) | Aliphatic index | Grand average of hydropathicity (GRAVY) | Ext. coefficient  |
| :---- | ----- | ----- | ----- | :---- | :---- | :---- | :---- | ----- | ----- | :---- | :---- | ----- | :---: | ----- | ----- |
|  |  |  | **C** | **H** | **N** | **O** | **S** |  | **Mammalian reticulocytes, in vitro** | **Yeast, in vivo** | **Escherichia coli, in vivo** |  |  |  |  |
| **PD101** | 11 | C85H145N27O22S2 | 85 | 145 | 27 | 22 | 2 | 281 | 30 Hours | 20 Hours  | 10 Hours | 27.3 | 83.0 | 0.385 | 1490 |
| PD102 | 11 | C92H151N27O22S2  | 92 | 151 | 27 | 22 | 2 | 294 | 1.1 Hours | 3 min | 2 min | 47.85 | 83.0 | 0.545 | 1490 |
| PD103 | 9.50 | C88H148N30O25S2  | 88 | 148 | 30 | 25 | 2 | 293 | 1.1 Hours | 3 min | 2 min | 56.98 | 58.50 | \-0.250 | 0 |
| PD104 | 9.53 | C100H174N24O29  | 100 | 174 | 24 | 29 | 0 | 327 | 5.5 Hours | 3 min | 2 min | 4.27 | 136.5 | 0.375 | 0 |
| PD105 | 8.43 | C99H170N24O31  | 99 | 170 | 24 | 31 | 0 | 324 | 5.5 Hours | 3 min | 2 min | 8.99 | 117.00 | \-0.025 | 0 |
| PD106 | 9.31 | C102H171N27O27S2  | 102 | 171 | 27 | 27 | 2 | 329 | 5.5 Hours | 3 min | 2 min | 8.99 | 92.5 | 0.225 | 125 |
| PD107 | 8.43 | C104H179N25O29  | 104 | 179 | 25 | 29 | 0 | 337 | 20 Hours | 30 min | 10 Hours | 21.01 | 146.00 | 0.320 | 0 |
| PD108 | 6.21 | C104H177N25O31  | 104 | 177 | 25 | 31 | 0 | 337 | 20 Hours | 30 min | 10Hours | 47.23 | 131.5 | \-0.065 | 0 |
| PD109 | 4.08 | C98H152N22O34S2  | 98 | 152 | 22 | 34 | 2 | 308 | 1.1 Hours | 3 min | 2 min | 90.74 | 78.00 | \-0.045 | 125 |
| PD110 | 10 | C102H165N25O24S2  | 102 | 165 | 25 | 24 | 2 | 318 | 4.4 Hours | 20 Hours | 10 Hours | 19.40 | 78.50 | 0.235 | 1490 |
| PD111 | 9.63 | C100H163N25O24S3  | 100 | 163 | 25 | 24 | 3 | 315 | 4.4 Hours | 20 Hours | 10 Hours | 19.40 | 78.50 | 0.440 | 1490 |
| PD112 | 7.91 | C88H153N23O27S6  | 88 | 153 | 23 | 27 | 6 | 297 | 4.4 Hours | 20 Hours | 10 Hours | 49.58 | 49.00 | 0.065 | 250 |
| PD113 | 9.87 | C109H185N27O26S2  | 109 | 185 | 27 | 26 | 2 | 349 | 4.4 Hours | 20 Hours | 10 Hours | 31.54 | 126.50 | 0.435 | 1490 |
| PD114 | 9.87 | C107H181N27O27S2  | 107 | 181 | 27 | 27 | 2 | 344 | 4.4 Hours | 20 Hours | 10 Hours | 57.27 | 112.00 | 0.185 | 1490 |
| PD115 | 9.63 | C104H175N25O26S2  | 104 | 175 | 25 | 26 | 2 | 332 | 1.3 Hours | 3 min | 3 min | 8.60 | 131.00 | 0.885 | 1490 |
| PD116 | 4.78 | C98H164N26O33  | 98 | 164 | 26 | 33 | 0 | 321 | 4.4Hours | 20 Hours | 10 Hours | 47.06 | 112.00 | \-0.150 | 0 |
| PD117 | 4.51 | C97H160N26O35  | 97 | 160 | 26 | 35 | 0 | 318 | 4.4 Hours | 20 Hours | 10 Hours | 51.31 | 92.5 | \-0.550 | 0 |
| PD118 | 4.49 | C100H157N25O33S2  | 100 | 157 | 25 | 33 | 2 | 317 | 1.1 Hours | 3 Hours | 2 Hours | 106.91 | 73.00 | \-0.170 | 125 |
| PD119 | 10.29 | C94H164N26O26  | 94 | 164 | 26 | 26 | 0 | 310 | 4.4 Hours | 20 Hours | 10 Hours | 2.64 | 117.00 | 0.505 | 0 |
| PD120 | 9.32 | C91H157N25O26S1  | 91 | 157 | 25 | 26 | 1 | 300 | 4.4 Hours | 20 Hours | 10 Hours | 11.13 | 117 | 0.825 | 0 |
| PD121 | 7.99 | C93H159N25O28S3  | 93 | 159 | 25 | 28 | 3 | 308 | 4.4 Hours | 20 Hours | 10 Hours | 35.42 | 102.5 | 0.730 | 125 |
| PD122 | 12.01 | C100H174N34O27  | 100 | 174 | 34 | 27 | 0 | 335 | 5.5 Hours | 3 min | 2 min | 76.74 | 131.5 | \-0.130 | 0 |
| PD123 | 12.02 | C101H178N34O26  | 101 | 178 | 34 | 26 | 0 | 339 | 5.5 Hours | 3 min | 2 min | 67.11 | 131.5 | \-0.150 | 0 |
| PD124 | 7.96 | C97H167N29O29S3  | 97 | 167 | 29 | 29 | 3 | 325 | 5.5 Hours | 3 min | 2 min | 71.55 | 97.5 | \-0.195 | 125 |
| PD125 | 8.86 | C89H153N23O27S2  | 89 | 153 | 23 | 27 | 2 | 294 | 100 Hour | 20 Hours | 10 Hours | \-7.01 | 112.00 | 0.655 | 1615 |
| PD126 | 8.02 | C79H140N22O27S2  | 79 | 140 | 22 | 27 | 2 | 270 | 100 Hours | 20 Hours | 10 Hours | \-2.93 | 97.37 | 0.337 | 125 |
| PD127 | 7.96 | C82H144N24O30S3  | 82 | 144 | 24 | 30 | 3 | 283 | 1.3 Hours | 3 min | 3 min | 1.48 | 58.50 | \-0.325 | 125 |

